web/src/components/studies/ApprovedResearcherView.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import Button from "../ui/Button";
+import styles from "./ApprovedResearcherView.module.css";
+import CreateStudyForm from "./CreateStudyForm";
+
+type ApprovedResearcherViewProps = {
+  username: string;
+};
+export default function ApprovedResearcherView(props: ApprovedResearcherViewProps) {
+  const { username } = props;
+  const [createStudyFormOpen, setCreateStudyFormOpen] = useState(false);
+  const handleCreateStudyClick = () => {
+    setCreateStudyFormOpen(true);
+  };
+  return (
+    <>
+      {createStudyFormOpen && <CreateStudyForm username={username} setCreateStudyFormOpen={setCreateStudyFormOpen} />}
+
+      <h2 className={styles["content-heading"]}>Your Studies</h2>
+      <div className={styles["content-wrapper"]}>
+        <div className={styles["create-study-container"]}>
+          {/* if IsUCLStaff && */}
+          <Button onClick={handleCreateStudyClick} size="small" className={styles["create-study-button"]}>
+            Create Study
+          </Button>
+        </div>
+        <div>
+          <table>
+            <thead>
```

Comment:
styling? is there a table component in ui-kit?

Reply:
nope, none in progress or planned
we styled the other one ourselves. I was going to wait until after the entra stuff to bother with styling